src/ogma/features/styles.ts

```diff
@@ -422,8 +422,7 @@ export class StylesViz {
         },
         edgeAttributes: {
           text: {
-            scale: 3.0,
```

Comment:
it seems like a deliberate choice to add this and the scaling: true flag. Do we know we added it in the first place? are we sure it won't create another bug in another situation?
if we are sure, I'm ok to merge !